instrumentation-api-incubator/src/main/java/io/opentelemetry/instrumentation/api/incubator/builder/internal/DefaultHttpServerInstrumenterBuilder.java

```diff
@@ -53,23 +52,26 @@ public final class DefaultHttpServerInstrumenterBuilder<REQUEST, RESPONSE> {
       httpAttributesExtractorBuilder;
   private final HttpSpanNameExtractorBuilder<REQUEST> httpSpanNameExtractorBuilder;
 
-  @Nullable private TextMapGetter<REQUEST> headerGetter;
+  private final TextMapGetter<REQUEST> headerGetter;
```

Comment:
there is a null check for this field, so I guess it should have the `@Nullable`?

Reply:
it's actually never null as of now - but will be once ktor is converted to use the builder

Reply:
so I've added @Nullable back